openedx/features/custom_fields/multiselect_with_other/db/fields.py

```diff
@@ -116,12 +116,10 @@ def to_python(self, value):
                 return MSFList(choices, list(value))
         return MSFList(choices, [])
 
-    def _check_other_max_length_attribute(self, **kwargs):
+    def _check_other_max_length_attribute(self):
```

Comment:
At line `# 147` we are using this function and passing `kwargs`, Is removing `**kwargs` enough. Will code work?

```
# Line # 147
errors.extend(self._check_other_max_length_attribute(**kwargs))
```

Reply:
Thanks, fixed it now.